dashboard/app/models/teacher_feedback.rb

```diff
@@ -33,6 +33,9 @@ class TeacherFeedback < ApplicationRecord
   belongs_to :script
   belongs_to :level
   belongs_to :teacher, class_name: 'User'
+  scope :latest, -> {find_by(id: maximum(:id))}
+  scope :latest_per_level, -> {find(group([:student_id, :level_id]).pluck('MAX(teacher_feedbacks.id)'))}
+  scope :by_teacher, ->(teacher_id) {where(teacher_id: teacher_id) if teacher_id.present?}
```

Comment:
I'm not very familiar with how scopes work -- could you verify that the queries generated look reasonable?

Reply:
The scope `scope :latest, -> {find_by(id: maximum(:id))}` is equivalent to
```
def self.latest
  find_by(id: maximum(:id))
end
```
I plan on adding tests to verify that the scopes are working as expected! I did test it locally as I was working